sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Infrastructure/EnvironmentReader.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Azure.Core;
+
+namespace Azure.Messaging.EventHubs.Stress;
+
+/// <summary>
+///   Allows for reading and parsing a set of environment information in the
+///   dotenv format.
+/// </summary>
+///
+/// <seealso href="https://www.dotenv.org/"/>
+///
+internal static class EnvironmentReader
+{
+    /// <summary>
+    ///   Loads a dotenv file and parses the contents.
+    /// </summary>
+    ///
+    /// <param name="filePath">The full path, including filename, to the dotenv file.</param>
+    ///
+    /// <returns>The set of environment content parsed from the file.</returns>
+    ///
+    /// <exception cref="ArgumentException">Occurs when <paramref name="filePath"/> is <c>null</c> or empty.</exception>
+    /// <exception cref="FileNotFoundException">Occurs when <paramref name="filePath" /> does not exist.</exception>
+    /// <exception cref="FormatException">Occurs when a line of the file is malformed or the file contains duplicate environment variables.</exception>
+    ///
+    public static Dictionary<string, string> LoadFromFile(string filePath)
+    {
+        Argument.AssertNotNullOrEmpty(filePath, nameof(filePath));
+
+        if (!File.Exists(filePath))
+        {
+            throw new FileNotFoundException($"The environment file was not found.in '{ filePath }'");
+        }
+
+        var environment = new Dictionary<string, string>();
+        var count = 0;
+
+        foreach (var fileLine in ReadFileLines(filePath))
+        {
+            ++count;
+
+            var line = fileLine.AsSpan();
+            var firstCharacterPos = FindFirstCharacterIndex(line);
+
+            if ((firstCharacterPos == -1) || (line[firstCharacterPos] == '#'))
+            {
+                continue;
+            }
+
+            var separator = line.IndexOf('=');
+
+            if (separator == -1)
+            {
+                throw new FormatException($"The environment file is malformed at line { count }: '{ line.ToString() }'");
+            }
+
+            try
+            {
+                var parsedKey = line.Slice(firstCharacterPos, separator).Trim().ToString();
+                var parsedValue = line.Slice(separator + 1).Trim().ToString();
+                if (environment.ContainsKey(parsedKey))
```

Comment:
Updated this, ref https://github.com/bolorundurowb/dotenv.net/blob/ecbfcaa9dda1883495c56043f242d97136018cb8/src/dotenv.net/Utilities/Helpers.cs#L37